src/components/RangeSlider/index.js

```diff
@@ -16,13 +16,28 @@ class RangeSlider extends Component {
     }
   }
 
+  roundLabelValue = value => {
```

Comment:
Should this function exist outside of the component in utils somewhere? So that it can be tested separately too?